client/clientd/src/main.rs

```diff
@@ -24,7 +26,8 @@ struct Config {
     workdir: PathBuf,
 }
 struct State {
-    client: Client<UserClientConfig>,
+    client: Arc<Client<UserClientConfig>>,
+    fetch_tx: Sender<()>,
```

Comment:
Let's call this `fetch_sender`.

`fetch_tx` sounds like something that's going to lookup a transaction.